src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/repository/LicenceRepository.kt

```diff
@@ -245,6 +245,18 @@ interface LicenceRepository :
     """,
   )
   fun findLicenceAndVariations(licenceId: Long): List<Licence>
+
+  @Query(
+    """
+      SELECT l
+      FROM Licence l
+      WHERE l.statusCode in (uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceStatus.IN_PROGRESS, uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceStatus.SUBMITTED, uk.gov.justice.digital.hmpps.createandvaryalicenceapi.util.LicenceStatus.APPROVED)
+      AND l.id > :lastUpdatedLicenceId
+      ORDER BY l.id ASC
+      LIMIT :numberOfLicences
+    """,
+  )
+  fun findLicencesToBatchUpdateLsd(numberOfLicences: Long, lastUpdatedLicenceId: Long?): List<Licence>
```

Comment:
super minor and not worth changing.. but generally order params by significance. 
lastUpdatedId is more significant for correctness and impacts the logic, whereas number of records is more "meta"